modules/swagger-servlet/src/main/java/io/swagger/servlet/extensions/ServletReaderExtension.java

```diff
@@ -405,8 +404,8 @@ public void applyImplicitParameters(ReaderContext context, Operation operation,
 
     @Override
     public void applyExtensions(ReaderContext context, Operation operation, Method method) {
-        final ApiOperation apiOperation = method.getAnnotation( ApiOperation.class );
-        if( apiOperation != null ) {
+        final ApiOperation apiOperation = method.getAnnotation(ApiOperation.class);
+        if (apiOperation != null) {
             operation.getVendorExtensions().putAll(BaseReaderUtils.parseExtensions(apiOperation.extensions()));
         }
     }
```

Comment:
All these cleanup should be part of separate PR

Reply:
I'm going to push this in another PR.